### DISCUSSION TITLE
Preferred ingest method for system logs (sysmon > osquery, wazuh, winlogbeat)

### DISCUSSION THREAD

**Discussion by metapolitics** [Security-Onion-Solutions/securityonion#2187] [Unsupported Versions] [answered]
Hi

Can I just confirm what the best method is for ingesting sysmon logs for windows? While OSQuery, Wazuh, and Winlogbeat can all pass the sysmon logs back into SO, I want to ensure its being correlated/processed correctly during ingestion. 

Based on the documentation, I am assuming the preferred method is via winlogbeat ?

**Comment by defensivedepth** [comment, marked as answer]
All three applications have full support for sysmon parsing - in fact, the way we do sysmon parsing is that we normalize the logs and then pass it off to the sysmon parser, so all three go through the same exact parsing.

So the preferred method is whichever one works best for your environment.
